taskmanager-api/src/main/java/com/cardinity/taskmanager/service/TaskService.java

```diff
@@ -126,8 +126,15 @@ public Task getTaskById(@NotNull long taskId){
     }
 
 
-    public List<Task> getTasks(){
-        List<Task> tasks = this.taskRepository.findAll();
+    public List<Task> getTasks(TaskStatus status) {
+        List<Task> tasks;
+        if (status != null) {
```

Comment:
This can be done with ExampleMatcher

Reply:
Created a commit where used `ExampleMatcher` removed method from the repository as well.